src/base/synchronization/auto_signaller.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <cstddef>
+#include <utility>
```

Comment:
move to .cc file, as the header file doesn't need it

Reply:
Done with required changes.